src/components/todo-list.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { getToDoList } from 'services/get-todo-list-service';
+import { ToDoCard } from 'components/todo-card';
+import * as S from '../styles/styled';
+
+export function ToDoList() {
+  const [toDos, setToDos] = React.useState<
+    { id: number; title: string; description: string }[]
+  >([]);
+
+  const todoList = async () => {
+    const response = await getToDoList();
+
+    setToDos(response);
+  };
+
+  React.useEffect(() => {
+    todoList();
+  }, []);
```

Comment:
pra declarar uma função dentro do componente e usar em um effect, é melhor que ela esteja dentro de um  useCallback, mas nesse caso que é simples, o melhor é resolver tudo dentro do effect de uma vez assim:

```suggestion
  React.useEffect(() => {
    const fetchTodoList= async () => {
    const responseTodoList = await getToDoList();
    setToDos(responseTodoList);
  };
    fetchTodoList();
  }, []);
```

fora isso, o nome todoList pra essa função ficou ruim, pois não descreve o que a função faz, fica parecendo que ela é uma const com a lista de todo já pronta... melhor seria algo tipo fetchTodoList ou getTodoList ou até handleGetTodoList